test/gracenote/register_test.rb

```diff
@@ -3,17 +3,25 @@
 module Gracenote
   describe "client" do
     describe "#register" do
-      it "should return the userid" do
-        stub_request(:post, //).to_return(
-          headers: { "Content-Type" => "applicaiton/xml" },
-          body: <<-XML
+      let(:response_xml) do
+        <<-XML
 <RESPONSES>
   <RESPONSE STATUS="OK">
     <USER>F3180SG7792B09ZPN7-UFC6XA8FV3RQ80K0RO7FURV76GQSDM16</USER>
   </RESPONSE>
 </RESPONSES>
-          XML
-        )
+        XML
+      end
+
+      it "sends client_id" do
+        stub_request(:post, //).to_return(headers: { 'Content-Type' => 'applicaiton/xml' } , body: response_xml)
+        client = Gracenote::Client.new(client_id: "client_id_string")
+        allow(client).to receive(:post).with(%r{<CLIENT>client_id_string</CLIENT>}).and_call_original
+        client.register
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Space found before comma.